### ISSUE TITLE
question about<Semantic Brightness Consistency Constraint loss>code 

### ISSUE THREAD

**Issue by JetsonKarl** [LingLIx/SCL-LLE#3]
Myloss.py line143:d2 = torch.mean(a2)

**Comment by JetsonKarl** [comment]
it can not represent equation 6. There are some zeros in the tensor but you have not deleted them. So it can not represent this area's average brightness level, the truly B is bigger than your result.

**Comment by LingLIx** [comment]
> 它不能表示方程 6。张量中有一些零，但您没有删除它们。所以它不能代表这个区域的平均亮度水平，真正的B大于你的结果。
Yes. You are correct. We will change this part of the code later. Thank you very much for asking this question.
